src/prediction_market/serializers/prediction_market_serializer.py

```diff
@@ -0,0 +1,47 @@
+from django.db.models import Sum
+from rest_framework.serializers import ModelSerializer, SerializerMethodField
+
+from prediction_market.models import PredictionMarket
+from researchhub.serializers import DynamicModelFieldSerializer
+
+
+class PredictionMarketSerializer(ModelSerializer):
+    class Meta:
+        model = PredictionMarket
+        fields = "__all__"
+
+
+class DynamicPredictionMarketSerializer(DynamicModelFieldSerializer):
+    votes = SerializerMethodField()
+    bets = SerializerMethodField()
+
+    class Meta:
+        model = PredictionMarket
+        fields = ["id", "prediction_type", "end_date", "status", "votes", "bets"]
+
+    def get_votes(self, obj):
+        total_votes = obj.prediction_market_votes.count()
```

Comment:
We should also probably store this value in the PredictionMarket class to effectively "cache" it for faster loads